common/source/docs/common-mavlink-mission-command-messages-mav_cmd.rst

```diff
@@ -924,6 +750,79 @@ using an airspeed sensor.
    </tbody>
    </table>
 
+.. _mav_cmd_nav_vtol_takeoff:
+
+
+MAV_CMD_NAV_VTOL_TAKEOFF
+------------------------
+
+Supported by:  Plane  (not Copter or Rover). Specifically QuadPlanes.
+
+Takeoff while in VTOL mode.
+
+
+QuadPlane
+~~~~~~~~~
+
+The vehicle will climb straight up from it’s current location to the
+specified altitude. If the mission is begun while the vehicle is already
+flying, the vehicle will climb straight up to the specified altitude, if
+the vehicle is already above the altitude the command will be ignored
```

Comment:
The default behaviour is that the altitude is relative to the altitude at the time the command starts. If Q_OPTIONS bit 3 is set then the altitude instead uses the frame type of the command.